oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -73,6 +72,7 @@ impl Configuration {
         &self,
         config_name: &str, // Used for pretty debugging
         runtime: RuntimeRef,
+        _noderef: NodeRef,
```

Comment:
What is this used for?

Reply:
When a Node is created and the runtime generates a new reference/id, i pass it here into the newly created threads so it knows its own reference. Currently unused as the runtime/channel methods don't use the references.